package.json

```diff
@@ -22,12 +22,13 @@
     "postcss-loader": "^6.2.1",
     "postcss-preset-env": "^8.0.1",
     "style-loader": "^3.3.1",
-    "ts-loader": "^9.2.6",
-    "typescript": "^4.5.5",
+    "ts-loader": "^9.5.4",
+    "typescript": "^5.9.2",
     "webpack": "^5.65.0",
     "webpack-cli": "^4.9.1"
   },
   "dependencies": {
+    "@hotwired/stimulus": "^3.2.2",
```

Comment:
question: is there a way to piggy back on the Wagtail-provided Stimulus? i.e. we use what core provides without a potential version divergence?

Ideally for React/Draftail too